src/hawkes/hawkes_process.jl

```diff
@@ -0,0 +1,169 @@
+struct HawkesProcess{T<:Real} <: AbstractPointProcess
+    μ::T
+    α::T
+    ω::T
+
+    function HawkesProcess(μ::T1, α::T2, ω::T3) where {T1,T2,T3}
+        any((μ, α, ω) .< 0) &&
+            throw(DomainError((μ, α, ω), "All parameters must be non-negative."))
+        T = promote_type(T1, T2, T3)
+        (μ_T, α_T, ω_T) = convert.(T, (μ, α, ω))
+        new{T}(μ_T, α_T, ω_T)
+    end
+end
+
+function Base.rand(rng::AbstractRNG, hp::HawkesProcess, tmin, tmax)
+    sim = simulate_poisson_times(rng, hp.μ, tmin, tmax) # Simulate Poisson process with base rate
+    sim_desc = generate_descendants(rng, sim, tmax, hp.α, hp.ω) # Recursively generates descendants from first events
+    append!(sim, sim_desc)
+    sort!(sim)
+    return History(sim, fill(nothing, length(sim)), tmin, tmax)
+end
+
+"""
+    StatsAPI.fit(rng, ::HawkesProcess, h::History; step_tol::Float64 = 1e-3, max_iter::Int = 1000)
+
+Expectation-Maximization algorithm from [E. Lewis, G. Mohler (2011)](https://api.semanticscholar.org/CorpusID:2105771)).
+The relevant calculations are in page 4, equations 6-13.
+
+Let (t_1 < ... < t_N) be the event times over the interval [0, T). We use the immigrant-descendant representation,
```

Comment:
If possible, one should refrain from using LaTeX in docstrings because it renders badly in the REPL. If you have to include formulas or symbols, it's better to put them inside backticks so that they display as monospace code, and then add a page to the Documenter.jl documentation for proper LaTeX equations

Reply:
You can visualize the docstring by typing
```
?fit
```
in a Julia REPL